### PR TITLE
[NRB-2024] NAIROBI 2024 - INITIAL PR

### PR DESCRIPTION
**Pull request by alvo254**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

### CODE REVIEW

**Comment by adrianmoisey** [comment]
As mentioned in the previous PRs ([here](https://github.com/devopsdays/devopsdays-web/pull/13513#issuecomment-1855358123)) the `yyyy-city` files are not needed and can be deleted.

File: data/events/2024-nairobi.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+description: "Devopsdays is coming to Nairobi!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2024-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2024-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate:  2024-06-01T08:00:00+00:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate:  2024-06-01T18:00:00+00:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (adrianmoisey) [inline data/events/2024-nairobi.yml:14]: Are these days correct?
Over in slack ([here](https://devopsdays.slack.com/archives/C05MNP1FV52/p1698737499689319)) the date mentioned was 6th April 2024

Comment (alvo254) [reply data/events/2024-nairobi.yml:14]: Yes, the dates are correct you will receive an update on slack.

File: data/events/2024-nairobi.yml [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Maamun Bwanakombo"
+    github: "https://github.com/Nabhany"
```
Comment (glasnt) [inline data/events/2024-nairobi.yml:53]: The `github` value just needs to be the GitHub username
```suggestion
    github: "Nabhany"
```

File: data/events/2024-nairobi.yml [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  - name: "Emma Nyakio"
+    github: ""
+    linkedin: "https://www.linkedin.com/in/emma-nyakio/"
+    image: ""
+  - name: "Lawrence Juma" 
+    github: ""
+    linkedin: ""
+    image: ""
+  - name: "Ivy Jeptoo"
+    github: "devopsdays"
```
Comment (glasnt) [inline data/events/2024-nairobi.yml:73]: This is our github :) If Ivy has an account, please add it
```suggestion
    github: ""
```

File: data/events/2024-nairobi.yml [hunk 4]
```diff
@@ -0,0 +1,104 @@
+name: "2024-nairobi" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "NAIROBI" # The displayed city name of the event. Capitalize it.
```
Comment (glasnt) [inline data/events/2024-nairobi.yml:3]: As a stylistic choice, we tend to have cities use Title case, not allcaps
```suggestion
city: "Nairobi" # The displayed city name of the event. Capitalize it.
```

File: data/events/2024-nairobi.yml [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+event_twitter: "devopsdaysnairobi" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
```
Comment (glasnt) [inline data/events/2024-nairobi.yml:4]: This account doesn't exist. I can see `devopsdaysNRB` is an account, is that the correct one?

File: data/events/2024-nairobi.yml [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (glasnt) [inline data/events/2024-nairobi.yml:51]: There's two more members of your team that haven't been added here, according to your emails.

**Review by glasnt — CHANGES_REQUESTED** [review]
There are a few updates that are needed, and an onboarding call (if not already organised).

**Comment by phrawzty** [comment]
> There are a few updates that are needed, and an onboarding call (if not already organised).

My understanding is that @adrianmoisey already onboarded them. Confirm / deny, Adrian?

**Comment by adrianmoisey** [comment]
> > There are a few updates that are needed, and an onboarding call (if not already organised).
> 
> My understanding is that @adrianmoisey already onboarded them. Confirm / deny, Adrian?

Correct, we've done the onboarding call.

File: data/events/2024-nairobi.yml (hunk 6 above)
Comment (alvo254) [reply data/events/2024-nairobi.yml:51]: Just one more member.

File: data/events/2024-nairobi.yml (hunk 1 above)
Comment (jyee) [reply data/events/2024-nairobi.yml:14]: Also, these times are set to +00:00, meaning they are GMT timezone. Nairobi is GMT+3, so you may want to update the dates to reflect that. e.g. `startdate: 2024-06-01T08:00:00+03:00`.

File: data/events/2024-nairobi.yml (hunk 6 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml:51]: It's two. We originally got 7 names on Aug 2, 2023, and an additional name on Mon, 11 Dec 2023.

We need the list of names we receive to match the pull request.

File: data/events/2024-nairobi.yml (hunk 1 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/f3ff9e0f5c6845efc64dd6d8e539333cb73da29d

File: data/events/2024-nairobi.yml (hunk 2 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f

File: data/events/2024-nairobi.yml (hunk 3 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f

File: data/events/2024-nairobi.yml (hunk 4 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f

File: data/events/2024-nairobi.yml (hunk 5 above)
Comment (adrianmoisey) [reply data/events/2024-nairobi.yml]: Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f

**Comment by adrianmoisey** [comment]
This PR is getting close.

At the moment there are issues with the team members.

From the emails we've received, it seems like the PR is missing:
- Alvin Ndungu
- Emma Nyakio

There is also an extra person in this pull request:
- Emmah Gachoki

Can you please reconcile this?

**Pull request merged**